assets/android.lua

```diff
@@ -2060,28 +2070,26 @@ local function run(android_app_state)
         return android.dl.dlopen(library, ffi_load)
     end
 
-    -- install native libraries into libs
-    local install = android.asset_loader("install")
-    if type(install) == "function" then
-        -- Don't uncompress assets if we don't have write permissions on external storage.
-        if not android.canWriteStorage() then
-            android.notification("Insuficient permissions", true)
-            ffi.C.sleep(1) -- to show notification
-            android.lib.ANativeActivity_finish(android.app.activity)
-            ffi.C.sleep(1) -- to do onPause -> onStop -> onDestroy.
-            error("can't work without rw external storage")
-        end
-        android.showProgress()
-        local start_time = os.time()
-        install()
-        local end_time = os.time()
-        android.dismissProgress()
-        local elapsed_time = os.difftime(end_time, start_time)
-        if elapsed_time > 0 then
-            android.LOGV(string.format("update installed in %d seconds", elapsed_time))
-        end
-    else
-        error("error loading install.lua")
+    -- Do not extract assets if we don't have write permissions on external storage.
+    if not android.canWriteStorage() then
+        android.notification("Insuficient permissions", true)
```

Comment:
Maybe something more like this?
```suggestion
        android.notification("Write permission required to extract resources.", true)
```

Or if you're purposefully being vague (which I'd prefer not to be), fix the typo ;-)
```suggestion
        android.notification("Insufficient permissions", true)
```

Reply:
this toast is shown when the user just rejected giving the app storage permissions, so the second suggestion should be sufficient for users. But the first one is good for the lua error that logcat throws. :+1: